x509-cert/test-support/src/zlint.rs

```diff
@@ -0,0 +1,195 @@
+use serde::{
```

Comment:
FYI, I snagged https://crates.io/crates/zlint

Might be interesting to extract this into a separate crate (after landing this PR perhaps)